validator/remote/src/main/java/tech/pegasys/teku/validator/remote/sentry/SentryValidatorApiChannel.java

```diff
@@ -159,6 +160,12 @@ public SafeFuture<Optional<InclusionList>> createInclusionList(
         .createInclusionList(slot, validatorIndex);
   }
 
+  @Override
+  public SafeFuture<Optional<List<Transaction>>> getInclusionList(final UInt64 slot) {
+    // TODO EIP7805 add a separate sentry node channel for ILs
```

Comment:
why would we need a sentry node channel for this? I thought sentry node was for block publishing..